shared/macros/10-bash.jinja

```diff
@@ -982,7 +982,24 @@ if [ -f /usr/bin/authselect ]; then
         {{{ bash_ensure_pam_module_line("$PAM_FILE_PATH", 'password', control, 'pam_pwhistory.so', after_match) | indent(8) }}}
     fi
 else
+{{% if 'ubuntu' in product %}}
+conf_name={{{ pam_file }}}
+conf_path="/usr/share/pam-configs"
+
+if [ ! -f "$conf_path"/"$conf_name" ]; then
+    cat << EOF > "$conf_path"/"$conf_name"
+Name: pwhistory password history checking
+Default: yes
+Priority: 1024
+Password-Type: Primary
+Password: {{{ control }}} pam_pwhistory.so remember=24 enforce_for_root try_first_pass use_authtok
```

Comment:
Should this be one line or
```
Password:
    {{{ control }}} pam_pwhistory.so remember=24 enforce_for_root try_first_pass use_authtok
```

Reply:
pam-auth-update can identify the non new-line conf, so no need to change this